zest/releaser/utils.py

```diff
@@ -433,12 +530,16 @@ def run_entry_points(which_releaser, when, data):
         plugin(data)
 
 
-def _execute_command(command, input_value=''):
+def _execute_command(command, input_value=u''):
     """commands.getoutput() replacement that also works on windows"""
-    logger.debug("Running command: %r", command)
-    if command.startswith(sys.executable):
+    if isinstance(command, six.string_types):
+        # BBB
+        command = shlex.split(command)
+
+    logger.debug(u"Running command: {0}".format(cmd_to_text(command)))
+    if command[0] == sys.executable:
         env = dict(os.environ, PYTHONPATH=os.pathsep.join(sys.path))
-        if ' upload ' in command or ' register ' in command:
+        if u'upload' in command or u'register' in command:
```

Comment:
This has changed because the argument has changed. It is now a list of command parts (which is then escaped properly) rather than a single string containing the full command (which may not be escaped properly).